_data/sidebars/home_sidebar.yml

```diff
@@ -46,6 +46,10 @@ entries:
       url: /basic-widgets/
       output: web
 
+    - title: Material Design widgets
```

Comment:
When documenting material design for the Android developer site, the official guidance that we received from Roman Nurik was to use lowercase "material design". This is consistent with usage in the external [Google spec](https://material.googleplex.com/#).